CONTRIBUTING.md

```diff
@@ -1,8 +1,9 @@
 # Contributing
 
 You may find [this live editor](https://usaco.guide/editor) helpful when editing
-modules. For more information contributing, view our
-[Contributing module](https://usaco.guide/general/contributing).
+modules. For more information about contributing, view our
+[Contributing module](https://usaco.guide/general/contributing) or our
+[docs](docs).
```

Comment:
shouldn't this link to github?

Reply:
it does, should i change it to an absolute link?

Reply:
Oh oops, didn't realize it was the github file. No, this is fine as is